tnefparse/cmdline.py

```diff
@@ -47,18 +46,13 @@
          help="extract a json dump of the tnef contents")
 
 
-def tnefparse() -> None:
+def tnefparse(argv=None) -> None:
     "command-line script"
 
-    if len(sys.argv) == 1:
-        parser.print_help()
-        sys.exit(1)
-
-    args = parser.parse_args()
+    args = parser.parse_args(argv)
 
     if args.logging:
-        level = eval("logging." + args.logging)
-        logger.setLevel(level)
+        logger.setLevel(getattr(logging, args.logging, logging.ERROR))
```

Comment:
Is this really the same behavior?

I def. should read up on Python / logging asap.

If `args.logging` is not set, what is the log level then? Beforehands it was `ERROR`.

From the [logging-docs](https://docs.python.org/3/howto/logging.html#when-to-use-logging):

> The default level is WARNING, which means that only events of this level and above will be tracked, unless the logging package is configured to do otherwise

Reply:
Not sure what you mean @jugmac00 ... if `args.logging` is not set, this code won't execute to begin with and the level stays as set in the beginning of module: `logger.setLevel(logging.ERROR)`. So LGTM?

Reply:
I replaced the default `setLevel` from the top into the default value within `argparse`, so we `setLevel` once only.

Reply:
Ah ok. Just in case , there's also this btw:

https://github.com/koodaamo/tnefparse/blob/77f327d5b97bb7eb6add392b52fe87fa2bb6c1c1/tnefparse/__init__.py#L5